Exporter/src/test/resources/init-full-export.sql

```diff
@@ -1,24 +1,44 @@
 -- User
 INSERT INTO User (activationKey, active, auxProfileInfo, email, fullName, keyTime, login, passwordDigest, secret, superuser, ssoSubject) VALUES (null, true, null, 'biostudies-dev@ebi.ac.uk', 'Biostudy manager', 0, 'manager', 0x9C22709B6EFF014C8D11A52E4EBA161CFC2D2DFC, '3bd53494-e672-41d4-b491-27f77bdb59b8', true, null);
 
+-- AccessTags
+INSERT INTO AccessTag (id, description, name, owner_id, parent_tag_id) VALUES (1, null, 'Public', 1, null);
+
 -- Submission
 INSERT INTO Submission (CTime, MTime, RTime, accNo, relPath, released, rootPath, title, version, owner_id, secretKey) VALUES (1460402396, 1460402396, 1460402396, 'S-EPMC2873748', 'S-EPMC/S-EPMCxxx748/S-EPMC2873748', true, 'S-EPMC/S-EPMCxxx748/S-EPMC2873748', 'Cost-effectiveness of a potential vaccine for human papillomavirus.', 1, 1, null);
+INSERT INTO Submission (CTime, MTime, RTime, accNo, relPath, released, rootPath, title, version, owner_id, secretKey) VALUES (1460389106, 1460389106, 1460389106, 'S-EPMC3343633', 'S-EPMC/S-EPMCxxx633/S-EPMC3343633', true, 'S-EPMC/S-EPMCxxx633/S-EPMC3343633', 'Transcription factor RORα is critical for nuocyte development.', 1, 1, null);
+
+-- Submission Access Tag
+INSERT INTO Submission_AccessTag (Submission_id, accessTags_id) VALUES (2, 1);
 
 -- Sections
+-- Submission 1
 INSERT INTO Section (accNo, global, parentAccNo, tableIndex, type, parent_id, submission_id, ord) VALUES (null, false, null, -1, 'Study', null, 1, null);
 INSERT INTO Section (accNo, global, parentAccNo, tableIndex, type, parent_id, submission_id, ord) VALUES ('PMC2873748', false, null, -1, 'Publication', 1, 1, 0);
 INSERT INTO Section (accNo, global, parentAccNo, tableIndex, type, parent_id, submission_id, ord) VALUES ('a1', false, null, -1, 'Author', 1, 1, 1);
 INSERT INTO Section (accNo, global, parentAccNo, tableIndex, type, parent_id, submission_id, ord) VALUES ('a2', false, null, -1, 'Author', 1, 1, 2);
 INSERT INTO Section (accNo, global, parentAccNo, tableIndex, type, parent_id, submission_id, ord) VALUES ('o1', false, null, -1, 'Organization', 1, 1, 3);
+-- Submission 2
+INSERT INTO Section (accNo, global, parentAccNo, tableIndex, type, parent_id, submission_id, ord) VALUES (null, false, null, -1, 'Study', null, 2, null);
```

Comment:
I will sugesst to add extra scripts, like `private_submission.sql` and  `public_submission.sql` and added in test in this way is clear the script puporses.